Source/DafnyLanguageServer/Plugins/IQuickFixInput.cs

```diff
@@ -0,0 +1,19 @@
+﻿using OmniSharp.Extensions.LanguageServer.Protocol.Models;
+
+namespace Microsoft.Dafny.LanguageServer.Plugins;
+
+public interface IQuickFixInput {
```

Comment:
Why not use `DafnyDocument` instead of this?

Reply:
I prefer to have an interface for the API rather than a concrete implementation that might be evolving.

Also, for a suitable API, we don't want to expose plugin developers with more information than necessary, do we?